easybuild/easyblocks/g/gcc.py

```diff
@@ -203,6 +203,13 @@ def __init__(self, *args, **kwargs):
             self.log.warning('Setting withnvptx to False, since we are building on a RISC-V system')
             self.cfg['withnvptx'] = False
 
+        # define list of subdirectories in search paths of module load environment
+        self.module_load_environment.PATH = ['bin']
+        self.module_load_environment.LD_LIBRARY_PATH = ['lib', 'lib64']
+        # GCC can find its own headers and libraries but the .so's need to be in LD_LIBRARY_PATH
+        self.module_load_environment.CPATH = []
+        self.module_load_environment.LIBRARY_PATH = ['lib', 'lib64'] if get_cpu_family() == RISCV else []
```

Comment:
The RISC-V exception was added here in #3256, we should check the impact of not including `lib`/`lib64` on a RISC-V system.

@bedroge Was this really necessary, or is it really only required during the build of GCC itself?

Reply:
Not 100% sure, but I think the final installation wouldn't work without this either.

Reply:
IMO leaving those paths in `LIBRARY_PATH` should not cause any harm beyond having two extra dirs in that environment variable. So I updated this easyblock to remove `CPATH` but leave `LIBRARY_PATH` with default paths: https://github.com/easybuilders/easybuild-easyblocks/pull/3584/files/04dc2b383f7116d218e7fe984ae47cd7ab063eed..9926576e2256d421d0a0cc768c2d6ca85d6b7119